pages/schedule.md

```diff
@@ -0,0 +1,156 @@
+title: Preliminary schedule
+---
+
+Here's a provisional list of the talks and workshops taking place on each day.  A full schedule will be out soon!
+
+## Thursday 26th October
+ * Breezy, a platform for experiments in version control (Martin Packman)
+ * Building Quart from Flask and Asyncio: Lessons learnt reimplementing Flask using asyncio (Phil)
+ * Building a python frontend for HPC codes (Alice Harpole)
+ * Building a real-time audio sampling app on the PyBoard: A talk exploring MicroPython, the PyBoard and the PyBoard audio module (Alan Christie)
+ * Docs or it didn't happen!: How you can learn to stop worrying and write the docs (Mikey Ariel)
+ * Fun With Serverless Applications (Lorna Mitchell)
+ * How to build a self-hosted CI/CD pipeline with opensource (Iacopo Spalletti)
+ * Implementing Custom Containers: No, not Docker, Python collections (Claus)
+ * Lessons from combing micro:bits raspberry pis and tensorflow (Luke Spademan)
+ * Polygons, hill climbing, metallurgy and why you are here: (no philosophical life advice given in this talk) (Vincent Knight)
+ * PostgreSQL: Beyond the Basics (Lorna Mitchell)
+ * Programming Music for Performance: Live coding with FoxDot (Ryan Kirkbride)
+ * Programming Music for Performance: Live coding with FoxDot (Ryan Kirkbride)
```

Comment:
That this is repeated twice is a little unfortunate without the talk/workshop distinction, but will do for now.